src/main/java/net/sf/jabref/importer/fileformat/MedlinePlainImporter.java

```diff
@@ -213,7 +214,7 @@ else if ("BTI".equals(lab) || "CTI".equals(lab)) {
                     if (oldAb == null) {
                         hm.put("abstract", val);
                     } else {
-                        hm.put("abstract", oldAb + "\n" + val);
+                        hm.put("abstract", oldAb + Globals.NEWLINE + val);
```

Comment:
No, every time JabRef writes a newline somewhere it uses Globals.NEWLINE...except here.